tests/gnmi/helper.py

```diff
@@ -96,6 +96,7 @@ def apply_cert_config(duthost):
     dut_command += "--config_table_name GNMI_CLIENT_CERT "
     dut_command += "--client_auth cert "
     dut_command += "--enable_crl=true "
+    dut_command += "--zmq_address=tcp://127.0.0.1:8100 "
```

Comment:
>zmq_address [](http://example.com/codeflow?start=22&length=11)

Is it possible that this function will be on a platform without zmq? If yes, we need to conditionally use `--zmq_address` parameter.

Reply:
Updated.